jacs-model-rendering/src/main/java/org/janelia/rendering/utils/ImageUtils.java

```diff
@@ -141,15 +141,23 @@ private static class PixelDataHandlers<B> {
         }
     }
 
+    /**
+     * Use the reference to {@code openSeekableStream} instead.
+     *
+     * @return
+     */
+    @Deprecated
     public static Function<Path, InputStream> getImagePathHandler() {
-        return (Path p) -> {
-            try {
-                return new FileSeekableStream(p.toFile());
-            } catch (IOException e) {
-                LOG.error("Error opening {}", p, e);
-                throw new IllegalArgumentException(e);
-            }
-        };
+        return ImageUtils::openSeekableStream;
+    }
```

Comment:
this method is not used anywhere anymore but this is what we used in the past versions